paper/src/main/java/net/william278/huskhomes/PaperHuskHomes.java

```diff
@@ -57,4 +75,63 @@ public List<Command> registerCommands() {
                 .collect(Collectors.toList());
     }
 
+    @Override
+    public int runAsync(@NotNull Runnable runnable) {
+        if (folia) {
+            return paperTaskRunner.runAsync(runnable); // For Folia
+        }
+        return super.runAsync(runnable); // For Paper and forks
+    }
+
+    @Override
+    public <T> CompletableFuture<T> supplyAsync(@NotNull Supplier<T> supplier) {
+        if (folia) {
+            return paperTaskRunner.supplyAsync(supplier);
+        }
+        return super.supplyAsync(supplier);
+    }
+
+    @Override
+    public void runSync(@NotNull Runnable runnable) {
+        if (folia) {
+            paperTaskRunner.runSync(runnable);
+        } else {
+            super.runSync(runnable);
+        }
+    }
+
+    @Override
+    public int runAsyncRepeating(@NotNull Runnable runnable, long period) {
+        if (folia) {
+            return paperTaskRunner.runAsyncRepeating(runnable, period);
+        }
+        return super.runAsyncRepeating(runnable, period);
+    }
+
+    @Override
+    public void runLater(@NotNull Runnable runnable, long delay) {
+        if (folia) {
+            paperTaskRunner.runLater(runnable, delay);
+        } else {
+            super.runLater(runnable, delay);
+        }
+    }
+
+    @Override
+    public void cancelTask(int taskId) {
+        if (folia) {
+            paperTaskRunner.cancelTask(taskId);
+        } else {
+            super.cancelTask(taskId);
+        }
+    }
+
+    @Override
+    public void cancelAllTasks() {
+        if (folia) {
+            paperTaskRunner.cancelAllTasks();
+        } else {
+            super.cancelAllTasks();
+        }
+    }
```

Comment:
This should be removed and PaperHuskHomes implement PaperTaskRunner instead :)

Reply:
I can't do it, you did the same when registering commands
![image](https://user-images.githubusercontent.com/70480001/235099941-5c8c7074-6ba9-439f-9543-fa6e215ed948.png)